data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+				`	"As long as I get paid for it."`
+			`	Xilin nods.`
+			`	"The Syndicate cohort will be waiting on Sunracer for the next one." He casts a look around, "With any luck you'll actually be able to get a drink around here by the time you return."`
+				accept
+
+
+event "mooncreek republic garrison"
+	system Cardax
+		add fleet "Small Republic" 1500
+		add fleet "Large Republic" 3000
+
+
+mission "HRF 07 Escort Syndicate Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Sunracer"
+	destination "Mooncreek"
+	on visit
+		dialog `You have arrived without the convoy you are escorting. You must take off and wait for it to continue.`
+	to offer
+		has "HRF 06 Escort Syndicate Outfitters: done"
+	on offer
+		event "mooncreek spaceport"
+		conversation
+			`When you arrive on Sunracer you are greeted by a Syndicate representative. They appear agitated by the delay and are ready to depart on their PR flight immediately.`
+				accept
+	npc accompany save
+		personality escort timid
+		government Syndicate
+		ship Freighter "S.S. Raleigh"
+		ship Freighter "S.S. Scott"
+		ship Freighter "S.S. Hudson"
+		ship Freighter "S.S. Drake"
+		ship Freighter "S.S. Magellan"
+		ship Freighter "S.S. Franklin"
+	npc accompany save
+		personality escort heroic
+		government Syndicate
+		fleet
+			names "syndicate capital"
+			variant
+				"Splinter (Mark II)" 2
+				"Manta (Mark II)" 3
+				"Quicksilver (Mark II)" 6
+				"Protector" 8
+
+
+event "mooncreek spaceport"
+	planet Mooncreek
+		attributes hai military north frontier
+		spaceport `The spaceport is essentially a military facility. Uniformed crew members of various stripes hurry back and forth without giving you a second glance. The food options are diverse, but all have that certain trend of solidity and mass over taste that is typically of military fare. There are few shops or other services catering to civilians.`
+		security 0.6
+
+
+mission "HRF 08 Escort Hai Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Mooncreek"
+	destination "Mountaintop"
+	passengers 1
+	to offer
+		has "HRF 07 Escort Syndicate Outfitters: done"
+	on offer
+		event "mooncreek syndicate garrison"
+		conversation
+			`The Syndicate convoy arrives to a more muted reception, but one that is thoroughly covered nonetheless. The rudimentary spaceport, still only days old, has now opened up a bar and the Lovelace outfitters have begun work on their facilities here. Hiyamaharu is waiting for you in the spaceport and appears to be sampling a collection of cheap whiskies.`
+			`	"Welcome back Captain," he says, "If you don't mind I'll be coming with you for the Hai escort."`
+			choice
+				`	"This seems mildly ridiculous."`
+				`	"Of course."`
+			`	"I hear the intent of this affair is to prevent the news cycle in human space from turning over before the big event," he says.`
```

Comment:
```suggestion
			`	"I hear the intent of this affair is to prevent the news cycle in human space from turning over before the big event," he explains.`
```